source/client-side-encryption/client-side-encryption.rst

```diff
@@ -819,10 +819,16 @@ EncryptOpts
 
 .. code:: typescript
 
+   enum QueryType {
+      QueryTypeEquality
```

Comment:
Is the prefix required? Could just `Equality` work?

Reply:
Done.